mpf/config_players/plugin_player.py

```diff
@@ -52,7 +52,8 @@ def register_player_events(self, config, mode=None, priority=0):
         events = super().register_player_events(config, mode, priority)
         # when bcp is disabled do not register plugin_player
         if self.machine.options['bcp']:
-            if self.machine.is_shutting_down:
+            # hasattr check is for IMC to work
+            if hasattr(self.machine, 'is_shutting_down') and self.machine.is_shutting_down:
```

Comment:
I know this has already merged, but you can combine the error-safe `hasattr` the truthiness of the value by passing a default value to `getattr`.

```
if getattr(self.machine, 'is_shutting_down', False):
  return events
```

Saves you one boolean evaluation. Hurray!

Reply:
Anthony, thanks!  I am learning Python here to help out, so I just learned about hasattr, but getattr certainly seems like a better fit.  Didn't even know that was a think before now!  I have a new PR I will push to get this adjusted while it is top of mind.